### PR TITLE
Remove temporary skip of tempest-all package

### PR DESCRIPTION
**Pull request by kopecmartin**
openstack-tempest-all package was temporarily replaced with only
openstack-tempest one because whitebox-neutron-tempest plugin
wasn't available on rhel.
These patches make sure that the plugin is included in the
openstack-tempest-all package only in upstream:
https://review.rdoproject.org/r/q/I93f6cee77927dfadd15c8093b49b1ed1f3e04cdc
Thus it's safe to get the things back as they were.

### CODE REVIEW

**Comment by viroel** [comment]
/approve

thanks, this is a must, we need plugins installed to continue with other tasks.

**Comment by gouthampacha** [comment]
+1

We'll need this to run tests from tempest plugins in the jobs... Thanks @kopecmartin

**Review by arxcruz — APPROVED** [review]
/approve but if it start to fails, I will revert again :)

**Comment by viroel** [comment]
> /approve but if it start to fails, I will revert again :)

It is worst to build image called `tempest-all` without `openstack-tempest-all` package, it causes confusion for those that are using it - it is better to stop building it or letting it failure until the correct fix is landed.

**Comment by kopecmartin** [comment]
recheck

**Pull request merged**
